x-pack/plugins/fleet/common/openapi/bundled.json

```
@@ -34,14 +34,16 @@
                 "schema": {
                   "type": "object",
                   "properties": {
-                    "name": {
-                      "type": "string"
-                    },
                     "status": {
                       "type": "string"
                     },
+                    "id": {
+                      "type": "string",
+                      "description": "fleet server host id"
```

```suggestion
                      "description": "Fleet Server host ID"
```
Just a small nit, but since it's the product name we should capitalize it. Same for the other instances.